IdentityCore/src/cache/MSIDKeychainTokenCache.m

```diff
@@ -37,7 +37,13 @@
 #import "NSKeyedArchiver+MSIDExtensions.h"
 #import "MSIDJsonObject.h"
 
-NSString *const MSIDAdalKeychainGroup = @"com.microsoft.adalcache";
+
+#if TARGET_OS_IPHONE
```

Comment:
Will this be a breaking change for existing MSAL apps?

Reply:
As discussed offline. This will not be a breaking change.